Catappult/appcoins-wallet-android
# Feature/eskl 348 eskills download install manager

**What does this PR do?**

   Download & Install Eskills game within the wallet AppView

**Database changed?**

   No

**How should this be manually tested?**
  1- E-skills AppView Download
      &ensp;1.1 - Click on Install Button
      &ensp;1.2 - If storage permissions are disabled, request permissions are displayed
      &ensp;1.3 - With permissions enabled, download should start with progress bar
      &ensp;1.4 - Test Pause , Resume and Cancel buttons
  2 - App-View Install
       &ensp;2.1 - When a download is finished, Install pop-up should be displayed
       &ensp;2.2 - Click on "Install" and check if progress bar is showing "Installing"
       &ensp;2.3 - When the installation is finished, the Install Button should be replaced with "Open"
  3 - Open App
      &ensp;3.1 - Test opening an app from the AppView

**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-348](https://aptoide.atlassian.net/browse/ESKL-348)

**Code Review Checklist**



[ESKL-348]: https://aptoide.atlassian.net/browse/ESKL-348?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
